build.gradle

```diff
@@ -1,50 +1,100 @@
+import cuchaz.enigma.command.ConvertMappingsCommand
+
+buildscript {
+    repositories {
+        maven {
+            name "Fabric Repository"
+            url 'https://maven.fabricmc.net'
+        }
+        mavenLocal()
+    }
+
+    dependencies {
+        classpath "cuchaz:enigma:0.15.0"
+    }
+}
+
 plugins {
     id 'maven'
     id 'maven-publish'
 }
 
+def localMappingsPath = "$buildDir/v2Mappings"
+mkdir(localMappingsPath)
```

Comment:
```suggestion
localMappingsPath.mkdir()
```

Reply:
seriously?

Reply:
🤔 what

Reply:
Oh, I missed the extra `s` at the end of `mkdirs`. My bad

Reply:
I realized my fault and added `s` :sweat:

Reply:
![image](https://user-images.githubusercontent.com/25990014/66872458-c0d45680-efae-11e9-887c-21abfadffa83.png)